### PR TITLE
DOC: Made minor change to fprime keyword description in fsolve

### PR DESCRIPTION
**Pull request by adivijaykumar**

### CODE REVIEW

File: scipy/optimize/minpack.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def fsolve(func, x0, args=(), fprime=None, full_output=0,
         The starting estimate for the roots of ``func(x) = 0``.
     args : tuple, optional
         Any extra arguments to `func`.
-    fprime : callable(x), optional
+    fprime : callable(x, *args), optional
```
Comment (dlax) [inline scipy/optimize/minpack.py]: This breaks documentation build. You can change to
```
callable ``f(x, *args)``
```
as it is for `func` argument above.

**Review by dlax — CHANGES_REQUESTED** [review]

File: scipy/optimize/minpack.py (hunk 1 above)
Comment (adivijaykumar) [inline scipy/optimize/minpack.py]: Oh right.

Just a doubt - is the way it was written initially ie. `callable(x)` a valid way of writing it then? Should that also have been ` callable ``f(x, *args)`` `?

Comment (dlax) [inline scipy/optimize/minpack.py]: I'd say you can make it consistent with `func` parameter so ` callable ``f(x, *args)`` `.

Comment (adivijaykumar) [inline scipy/optimize/minpack.py]: Sorry, I meant ` callable` ``f(x) `, but I think the doubt is solved anyway.

**Review by dlax — APPROVED** [review]

**Pull request merged**